app/src/main-process/menu/build-default-menu.ts

```diff
@@ -211,7 +211,7 @@ export function buildDefaultMenu(sharedProcess: SharedProcess): Electron.Menu {
         click: emit('open-in-shell'),
       },
       {
-        label: __DARWIN__ ? 'Show in Finder' : '&Show in Explorer',
+        label: __DARWIN__ ? 'Show in Finder' : 'Show in &Explorer',
```

Comment:
Sorry to mislead you earlier.  I hadn't tested my suggesting.  &E also conflicts.  
But &X works 👍